pytumblr/__init__.py

```diff
@@ -548,7 +548,7 @@ def _send_post(self, blogname, params):
 
         return self.send_api_request("post", url, params, valid_options)
 
-    def send_api_request(self, method, url, params={}, valid_parameters=[], needs_api_key=False):
+    def send_api_request(self, method, url, params={}, valid_parameters=[], needs_api_key=False, use_body=False):
```

Comment:
while i'm not exactly sure what to do here, this feels a bit weird for a few reasons:

1. this is a parameter that is only really available when `method == "delete"`
1. this means that we cannot use both query parameters and body parameters

i think the ideal here would be to have two separate arguments, `query_params` and `body_params` so we have the ability to send both. its not following any best practices to go that route, but it does cover all the bases so we can handle oddities that exist in various legacy api routes.